tests/test_data/test_datasets.py

```diff
@@ -301,11 +322,22 @@ def operation(arr):
     assert result.name == ds_name
 
 
+@pytest.mark.parametrize("dataset_type_ind", [0, 1])
 @pytest.mark.parametrize("ds_name", ["test123", None])
-def test_tetrahedral_dataset(log_capture, tmp_path, ds_name, no_vtk=False):
+def test_tetrahedral_dataset(log_capture, tmp_path, ds_name, dataset_type_ind, no_vtk=False):
```

Comment:
I'd change making this index variables more verbose for clarity, but it's a stylistic preference anyway so not that important.